basalt/utils/networker.py

```
@@ -64,3 +65,56 @@ def fetch(
 
         except Exception as e:
             return NetworkBaseError(str(e)), None
+            
+    async def async_fetch(
+            self,
+            url: str,
+            method: str,
+            body = None,
```

Flake8 (7.2.0)

[error] unexpected spaces around keyword / parameter equals

(E251)